keyboards/keebio/iris/keymaps/mnil/config.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#define EE_HANDS
+
+#undef RGBLED_NUM
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 12
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define RGBLIGHT_SPLIT
```

Comment:
You may want this instead? 
```suggestion
#define RGBLED_SPLIT { 6, 6 }
```

This implies the split setting, and specifies how many leds are on each half.